misk/src/main/kotlin/misk/tasks/RepeatedTaskQueueConfig.kt

```diff
@@ -0,0 +1,33 @@
+package misk.tasks
+
+import misk.backoff.Backoff
+import misk.backoff.ExponentialBackoff
+import java.time.Duration
+
+data class RepeatedTaskQueueConfig(
+  /**
+   * The default amount of jitter to use when scheduling backoffs.
+   *
+   * Can be overridden when scheduling a tasks.
+   */
+  val default_jitter_ms: Long = 50,
```

Comment:
is this just for tests?

Reply:
well i'm using it for tests, but it seems reasonable to make it configurable